app/views/posts/index.html.erb

```diff
@@ -1 +1,14 @@
-<h1>Here is a list of posts made by the same user</h1>
\ No newline at end of file
+<%= render 'shared/user', user: @user%>
+<ul>
+  <% if @posts.empty?%>
+    <p class='post-card-no-click'>No posts</p>
+  <% else%>
+  <div class='users-bio'><%=@user.bio%></div>
+  <% @posts.each do |post|%>
+    <li>
+      <%= render 'shared/post', post: post%>
+    </li>
+  <%end%>
+<%end%>
+</ul>
+<%= link_to 'Back', :back, class: 'back-btn btn-secondary' %>
```

Comment:
- [ ] Kindly ensure that you use the `recent_5_comments` in your post model to render 5 most recent comments in the app/views/posts/index.html.erb page and not the show.html.erb as per requirements

Wireframe
![image](https://github.com/lorenzoworx/velvetScribe/assets/81932478/661c3ece-c8f4-4216-ae4b-bd1891df8ab6)


Your design
![image](https://github.com/lorenzoworx/velvetScribe/assets/81932478/57c3e9a8-949f-44c3-9721-6b8601d542f5)


Notice the missing 5 recent comments